### PR TITLE
Docs/developer tutorials

### PR DESCRIPTION
**Pull request by NatalieEO**
Update developer tutorials

### CODE REVIEW

File: _docs/getting-started-with-cwl.md [hunk 1]
```diff
@@ -265,11 +267,11 @@ So what's going on here?  What's the Dockstore CLI doing?  It can best be summed
...
 
 **Tip:** you can use `--debug` to get much more information during this run, including the actual call to cwltool (which can be super helpful in debugging).
 
-**Tip:** the `dockstore` CLI automatically create a `datastore` directory in the current working directory where you execute the command and uses it for inputs/outputs.  It can get quite large depending on the tool/inputs/outputs being used.  Plan accordingly e.g. execute the dockstore CLI in a directory located on a partition with sufficient storage.
+**Tip:** the `dockstore` CLI automatically creates a `datastore` directory in the current working directory where you execute the command and uses it for inputs/outputs.  It can get quite large depending on the tool/inputs/outputs being used.  Plan accordingly e.g. execute the dockstore CLI in a directory located on a partition with sufficient storage.
```
Comment (coverbeck) [inline _docs/getting-started-with-cwl.md]: `e.g.` should generally be preceded with a comma.

File: _docs/getting-started-with-wdl.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ This downloads to my current directory.  I could choose another location, it rea
 }
 ```
 
-**Tip:** the Dockstore CLI can handle inputs at HTTPS, FTP, and S3 URLs but that's beyond the scope of this tutorial.
+**Tip:** the Dockstore CLI can handle inputs with HTTPS, FTP, and S3 URLs but that's beyond the scope of this tutorial.
```
Comment (coverbeck) [inline _docs/getting-started-with-wdl.md]: As of 1.6, we can also say GS urls. Not sure if it's worth putting in though.

File: _docs/workflows.md [hunk 3]
```diff
@@ -29,22 +29,22 @@ When Dockstore was created, CWL was the first descriptor language we supported.
...
 
-The first step is to create a CWL or WDL workflow descriptor for your workflow and then check it into GitHub, Bitbucket or GitLab in a repo, we recommend the filename `Dockstore.cwl` at the root of your repository for simplicity but anything else with a consistent extension should work just as well. The details as to how to write a workflow are somewhat beyond the scope of this tutorial but we can recommend the [Introduction to the CWL](http://www.commonwl.org/user_guide/) and [Getting Started with WDL](https://github.com/openwdl/wdl/tree/master#getting-started-with-wdl).
+The first step is to create a CWL or WDL workflow descriptor for your workflow and then check it into GitHub, Bitbucket or GitLab in a repo, we recommend the filename `Dockstore.cwl` at the root of your repository for simplicity but anything else with a consistent extension should work just as well. The details as to how to write a workflow are somewhat beyond the scope of this tutorial, but we can recommend the [Introduction to the CWL](http://www.commonwl.org/user_guide/) and [Getting Started with WDL](https://github.com/openwdl/wdl/tree/master#getting-started-with-wdl).
```
Comment (coverbeck) [inline _docs/workflows.md]: This first sentence seems like a run-on

File: _docs/register-on-dockstore.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ Before you can publish your new tool, you need to create a Dockstore account and
 
 *Only available in 1.5.0*
```
Comment (garyluu) [inline _docs/register-on-dockstore.md]: Might need to remove this.

File: _docs/workflows.md [hunk 5]
```diff
@@ -64,19 +64,23 @@ Now that you have linked your credentials and refreshed, there should be one stu
...
+
+![Register Workflow Manual2](/assets/images/docs/register_workflow_manual2.png)
+
+Source Code Provider allows you to choose between GitHub, BitBucket, and GitLab (your respective accounts for these third party repositories need to be linked to your Dockstore account). The Source Code Repository field must be filled out and is in the format `namespace/name` (the two paths may differ). The Workflow (descriptor) path and test parameter path are relative to the root of the Source Code Repository (and must begin with '/'). These will be the default locations to find their corresponding files, unless specified otherwise in the tags. The Workflow Name is an optional 'suffix' appended to the Dockstore path. It allows for two workflows to share the same Git paths; the Workflow Name uniquely distinguishes workflow repositories in Dockstore.
```
Comment (denis-yuen) [inline _docs/workflows.md]: Not within the scope of the ticket, but @garyluu @agduncan94 
Is there a reason why the "Add tag" button and "close" are left aligned in assets/images/docs/tageditor_modal.png now whereas  the button is on the right for "Register Tool", "Register Workflow", "Create Organization Request", etc.?

Comment (garyluu) [reply _docs/workflows.md]: just forgotten probably

**Pull request merged**